src/games/strategy/triplea/ai/proAI/util/ProSortMoveOptionsUtils.java

```diff
@@ -54,7 +54,8 @@ public int compare(final Map.Entry<Unit, Set<Territory>> o1, final Map.Entry<Uni
   }
 
   public static Map<Unit, Set<Territory>> sortUnitNeededOptions(final PlayerID player,
-      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap) {
+      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap,
+      final ProOddsCalculator calc) {
     final GameData data = ProData.getData();
 
     final List<Map.Entry<Unit, Set<Territory>>> list =
```

Comment:
Yeah, I've done this mostly manually so far. The issue is it sort of goes in cycles. I'll optimize it then I'll add a bunch of additional functionality or major refactor then needs to be optimized again. Sort of a never ending cycle when development is on going.